common/on_action/TFE_title_on_actions.txt

```diff
@@ -103,6 +106,9 @@ TFE_on_title_gain = {
 					set_de_jure_liege_title = title:e_rashidun
 				}
 			}
+			title:k_arabia = {

```

Comment:
Depending on converter's title mappings, `k_arabia` may be missing. Maybe wrap this in an
```
if = {
	limit = {
		exists = title:k_arabia
	}
	<effects>
}
```
block?

Reply:
Sure, done.